crates/client/rpc-core/src/lib.rs

```diff
@@ -145,4 +146,25 @@ pub trait StarknetRpcApi {
     /// contract.
     #[method(name = "getProof")]
     fn get_proof(&self, get_proof_input: RpcGetProofInput) -> RpcResult<RpcGetProofOutput>;
+
+    /// Simulate a given sequence of transactions on the requested state, and generate the execution
+    /// traces. If one of the transactions is reverted, raises error.
+    /// Spec version 0.4.0-rc2
```

Comment:
don't specify that plz

Reply:
You mean spec version or entire docstring?